tedana/__init__.py

```diff
@@ -36,6 +36,6 @@
          version=__version__, path='tedana', cite_module=True)
 
 # Citation for package version.
-due.cite(Doi('10.5281/zenodo.1250562'),
-         description='The tedana package version 0.0.1',
-         version='0.0.1', path='tedana', cite_module=True)
+due.cite(Doi('10.5281/zenodo.1289081'),
```

Comment:
Is there a way to just cite the general package, without specific versions ? Or do we need to update this every version release ? If the latter, we should star putting together a "version release" checklist, to make sure items like this aren't missed !

Reply:
We can use the Zenodo DOI for the general package and drop the version argument, but I think it makes more sense to keep this up to date with the version releases. I'm not sure how to do that, since the Zenodo DOI is generated only after we have made the release, but I'm sure someone has dealt with this issue before.

If we do use the version-specific DOI here and in the README, it might make it easier for us to track version-specific usage, assuming we can figure out how to track Zenodo DOI citations in the literature.

A "version release" checklist is a great idea, although I don't know what actually goes into the releases.

Reply:
I think fMRIPrep keeps [theirs in their wiki](https://github.com/poldracklab/fmriprep/wiki/Release-checklist), although I had shut down the wiki since I wanted to make sure information made it on to [our RTD site](https://tedana.readthedocs.io), first. Do you think it's something we could have as a section in the "Contributing" section of RTD ? Happy to handle this in another PR !